### PR TITLE
docs: remove unrelated git instructions

### PR DESCRIPTION
**Pull request by NickVolynkin**
We really don't need to teach people how to use git.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,17 +153,11 @@ python3 scripts/proof_tools.py push --bid_key <key of the bid> --ask_key <key of
 # Common issues
 
 ## Compilation Errors
```
Comment (alexandra-mara) [inline README.md:155]:
```suggestion
## Compilation errors
```

File: README.md [hunk 2]
```diff
@@ -153,17 +153,11 @@ python3 scripts/proof_tools.py push --bid_key <key of the bid> --ask_key <key of
...
-If you have more than one compiler installed i.e g++ & clang++. The make system might pick up the former. You can explicitly force usage of 
-clang++ by finding the path and passing it in the variable below.
+
+If you have more than one compiler installed, for example, g++ & clang++, the `cmake` might pick up the former.
+You can explicitly force usage of clang++ by finding the path and passing it in the variable below:
```
Comment (alexandra-mara) [inline README.md:158]:
```suggestion
You can explicitly force usage of clang++ by finding the path and passing it in the variable like this:
```

**Review by alexandra-mara — CHANGES_REQUESTED** [review]
Left a couple of suggestions

File: README.md [hunk 3]
```diff
@@ -153,17 +153,11 @@ python3 scripts/proof_tools.py push --bid_key <key of the bid> --ask_key <key of
 # Common issues
 
 ## Compilation Errors
-If you have more than one compiler installed i.e g++ & clang++. The make system might pick up the former. You can explicitly force usage of 
-clang++ by finding the path and passing it in the variable below.
+
+If you have more than one compiler installed, for example, g++ & clang++, the `cmake` might pick up the former.
```
[the remark this answers is not in the document]
Comment (NickVolynkin) [reply README.md:157]: why so?

[the remark this answers is not in the document]
Comment (alexandra-mara) [reply README.md]: No article is needed for a command-line tool. Only if it's further described like the X script/utility/tool etc. Like in [the official documentation](https://cmake.org/cmake/help/latest/manual/cmake.1.html), cmake, but the cmake executable

**Review by alexandra-mara — APPROVED** [review]
Lgtm

**Pull request merged**